Bottom: 08452cef620327a78c0c1bae781a9e2d9b1a977a
Top:    59eee8cb38d8714aac1aff782468541227a70cf4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 14:24:35 +0100

Refresh of generate-pcb-menu-res-and-gpcb

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 7efb2d7..21a9f5a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -230,8 +230,8 @@ EXTRA_DIST= \
 	$(srcdir)/hid/png/hid.conf \
 	$(srcdir)/hid/nelma/hid.conf \
 	$(srcdir)/hid/ps/hid.conf \
-	gpcb-menu.res \
-	pcb-menu.res \
+	gpcb-menu.res.in \
+	pcb-menu.res.in \
 	pcbtest.sh.in \
 	dbus.xml
 
@@ -239,12 +239,20 @@ AM_YFLAGS=	-d
 
 all-local: pcbtest.sh
 
+pcb-menu.res : pcb-menu.res.in
+	echo '/* AUTOMATICALLY GENERATED FROM pcb-menu.res.in DO NOT EDIT */' > $@
+	cat ${srcdir}/pcb-menu.res.in >> $@
+
 pcb-menu.h : pcb-menu.res
 	echo '/* AUTOMATICALLY GENERATED FROM pcb-menu.res DO NOT EDIT */' > $@
 	echo 'const char *pcb_menu_default[] = {' >> $@
 	sed 's/\\/\\\\/g; s/"/\\"/g; s/^/"/; s/$$/",/' < ${srcdir}/pcb-menu.res >> $@
 	echo '0};' >> $@
 
+gpcb-menu.res : gpcb-menu.res.in
+	echo '/* AUTOMATICALLY GENERATED FROM gpcb-menu.res.in DO NOT EDIT */' > $@
+	cat ${srcdir}/gpcb-menu.res.in >> $@
+
 gpcb-menu.h : gpcb-menu.res
 	echo '/* AUTOMATICALLY GENERATED FROM gpcb-menu.res DO NOT EDIT */' > $@
 	echo 'const char *gpcb_menu_default[] = {' >> $@
